connector/src/test/scala/com/microsoft/kusto/spark/KustoSourceE2E.scala

```diff
@@ -94,7 +95,7 @@ class KustoSourceE2E extends FlatSpec with BeforeAndAfterAll {
     val expectedNumberOfRows: Int =  100
     val rows: immutable.IndexedSeq[(String, Int)] = (1 to expectedNumberOfRows).map(v => (newRow(), v))
     val dfOrig = rows.toDF("name", "value")
-    val table = KustoQueryUtils.simplifyName(s"KustoSparkReadWriteTest_${UUID.randomUUID()}")
+    val table = "KustoSparkReadWriteTest_d8471fa5_93fc_4d0a_9ee0_74a4e994d114"
```

Comment:
Don't we want to generate a new table using a UUID?

Reply:
ill revert this file